go.mod

```diff
@@ -9,7 +10,9 @@ require (
 )
 
 require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
 	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/google/go-github/v29 v29.0.2 // indirect
```

Comment:
Oh!  I think I understand now... this is being pulled in because of the new dependency of `github.com/bradleyfalzon/ghinstallation` on line 4.

Hmmm... I'm not sure we actually want to do this in that case.  Let me think about it.

Reply:
Instead of adding this example to this repo, what would you think of adding it to the `github.com/bradleyfalzon/ghinstallation` repo?

I'm wondering if that would make more sense than adding it here, since this example is very specific to that package.

Reply:
Ah the woes of modularity! Wondering if we can update the version `github.com/bradleyfalzon/ghinstallation` uses to V41 https://github.com/bradleyfalzon/ghinstallation/blob/master/go.mod#L1. Right now it looks like they're using version 40. Would it be ok that we upgraded to version 40?

Reply:
Looks like it resolved v39 instead of 40, but at least thats better than v29.